exercise_10.js

```diff
@@ -0,0 +1,11 @@
+/*
+Complete the method that takes a boolean value and return a "Yes" string for true, or a "No" string for false.
+
+
+ */
+function boolToWord( bool ){
+    if (bool === true) {
```

Comment:
the syntax for if statement is as follows
```
if(<some expression that evaluates to to truth or falsy>) {
  <code to be run if the condition is met>
}
```
`<some expression that evaluates to truth or falsy>` - remember that it can be any expression. For example
```
if(1 > 2) {
  the condition is met
}
if({}) {
  the condition is met, because {} is truthy
}
if(true) {
  the condition is met because true is truthy
}
if(true === false) {
  the condition not is met because true is not equal false
}
if(!false) {
  the condition is met because !false evaluates to true, which is truthy
}
const amIAdult = true;
if(amIAdult === false ) {
  the condition is not met because amIAdult is not equal false, so the whole expression evaluates to false
}
if(amIAdult === true ) {
  the condition is met because amIAdult is equal true, so the whole expression evaluates to true
}

if(amIAdult) {
  the condition is met because the whole expression which is ONLY amIAdult usage have a true value
}

if(!amIAdult) {
  the condition is not met because the whole expression which is ONLY amIAdult with negation evaluates to false (not true equals false)
}
```
So if you have a boolean value you don't need to compare it against boolean.
`true === true` -> this evaluates to true
`false === false` ->  this evaluates to false